examples/tracing-compare.cpp

```diff
@@ -167,25 +169,20 @@ int main(int argc, char **argv) {
         });
   }
 
-  std::vector<TraceEvent> allEvents;
-
-  allEvents.push_back({"thread_name", 0, 'M', 0, {{"name", "CPU"}}});
-  allEvents.push_back({"thread_name", 0, 'M', 1, {{"name", "Interpreter"}}});
-#if (GLOW_WITH_OPENCL)
-  allEvents.push_back({"thread_name", 0, 'M', 2, {{"name", "OpenCL"}}});
-#endif
+  TraceContext allEvents(TraceLevel::STANDARD);
+  for (unsigned i = 0; i < supportedBackends.size(); ++i) {
```

Comment:
Mega nit: `for (unsigned i = 0, e = supportedBackends.size(); i < e; ++i) {`

I doubt we'll ever have enough backends for this to be a problem but should `i` be a `size_t` or maybe a `vector<T>::size_type`? You could also use a range based for with a separate counter.